bundles/org.openhab.binding.saicismart/src/main/java/org/openhab/binding/saicismart/internal/SAICiSMARTHandler.java

```diff
@@ -0,0 +1,267 @@
+                    sendACCommand((byte) 5, (byte) 8);
+                } catch (URISyntaxException | ExecutionException | TimeoutException | InterruptedException e) {
+                    logger.error("A/C On Command failed", e);
+                }
+            }).start();
+        } else if (channelUID.getId().equals(CHANNEL_SWITCH_AC) && command == OnOffType.OFF) {
+            // reset channel to off
+            updateState(CHANNEL_SWITCH_AC, OnOffType.OFF);
+            new Thread(() -> {
+                // disable air conditioning
+                try {
+                    sendACCommand((byte) 0, (byte) 0);
+                } catch (URISyntaxException | ExecutionException | TimeoutException | InterruptedException e) {
+                    logger.error("A/C Off Command failed", e);
+                }
+            }).start();
+        } else if (channelUID.getId().equals(CHANNEL_LAST_ACTIVITY) && command instanceof DateTimeType) {
+            // update internal activity date from external date
+            notifyCarActivity(((DateTimeType) command).getZonedDateTime(), true);
+        }
+    }
+
+    protected @Nullable SAICiSMARTBridgeHandler getBridgeHandler() {
+        return (SAICiSMARTBridgeHandler) super.getBridge().getHandler();
+    }
+
+    @Override
+    public void initialize() {
+        config = getConfigAs(SAICiSMARTVehicleConfiguration.class);
+
+        ThingBuilder thingBuilder = editThing();
+        thingBuilder.withChannels(thing.getChannels().stream().filter(c -> {
+            // TODO: check properties
+            return !c.getUID().getId().equals(CHANNEL_WINDOW_SUN_ROOF);
+        }).collect(Collectors.toList()));
+        updateThing(thingBuilder.build());
+
+        updateStatus(ThingStatus.UNKNOWN);
+
+        // just started, make sure we start querying
+        notifyCarActivity(ZonedDateTime.now(), true);
+
+        pollingJob = scheduler.scheduleWithFixedDelay(() -> {
+            if (lastCarActivity.isAfter(ZonedDateTime.now().minus(10, ChronoUnit.MINUTES))) {
+                if (this.getBridgeHandler().getUid() != null && this.getBridgeHandler().getToken() != null) {
+                    try {
+                        OTA_RVMVehicleStatusResp25857 otaRvmVehicleStatusResp25857 = new VehicleStateUpdater(this)
+                                .call();
+                        OTA_ChrgMangDataResp otaChrgMangDataResp = new ChargeStateUpdater(this).call();
+
+                        if (config.abrpUserToken != null && config.abrpUserToken.length() > 0) {
+                            String execute = ABRP.updateAbrp("8cfc314b-03cd-4efe-ab7d-4431cd8f2e2d",
+                                    config.abrpUserToken, otaRvmVehicleStatusResp25857, otaChrgMangDataResp);
+
+                            logger.debug("ABRP: {}", execute);
+                        }
+                    } catch (Exception e) {
+                        logger.warn("Could not refresh car data for {}. {}", config.vin, e.getMessage());
```

Comment:
```suggestion
                        logger.debug("Could not refresh car data for {}. {}", config.vin, e.getMessage());
```
As the thing is allready going offline, the log line is only used for aditional info and that is in this case for debugging